KsApi/models/User.swift

```diff
@@ -67,14 +68,14 @@ extension User: CustomDebugStringConvertible {
 }
 
 extension User: Decodable {
-  public static func decode(json: JSON) -> Decoded<User> {
+  public static func decode(_ json: JSON) -> Decoded<User> {
     let create = curry(User.init)
     let tmp = create
       <^> json <| "avatar"
       <*> json <|? "facebook_connected"
       <*> json <| "id"
       <*> json <|? "is_friend"
-      <*> json <|? "location"
+      <*> (json <|? "location" <|> .success(nil))
```

Comment:
this is the subtle argo change that i mentioned in the description of this PR. check it out and lemme know if you have q's.